pkg/capture/capture.go

```diff
@@ -180,7 +180,7 @@ func (cmd captureCommandRotate) execute(c *Capture) stateFn {
 	var result rotateResult
 
 	if c.flowLog.Len() == 0 {
-		logger.Debug("there are currently no flow records available")
+		logger.Infof("there are currently no flow records available on %s", c.iface)
```

Comment:
All good with Info. But remember, the interface is already logged as a structured field for every log line in capture (since it's put into the capture's context on instantiation and the logger is derived off that context).